Dockerfile.tmpl

```diff
@@ -205,13 +197,12 @@ RUN pip install pysal \
 
 RUN apt-get install -y libfreetype6-dev && \
     apt-get install -y libglib2.0-0 libxext6 libsm6 libxrender1 libfontconfig1 --fix-missing && \
+    # b/198300835 kornia 4.1.0 is not compatible with our version of numpy.
```

Comment:
Why was this comment added? Looks like `kornia` is unpinned now.

Reply:
I believe this was brought in my a conflict merge from main, I'll remove it.